### DISCUSSION TITLE
Set default market

### DISCUSSION THREAD

**Discussion by Vortilion** [on3iro/aeons-end-randomizer#444] [General]
Hi,

vielleicht bin ich blind, aber kann man den default market nicht ändern?

VG

**Comment by on3iro** [comment]
@Vortilion Hi, ja das aktuell tatsächlich nicht möglich. Ich erstelle mal ein Issue - aktuell habe ich leider wenig Kapazitäten, um an der App zu entwickeln.

**Reply by Vortilion** [reply]
Alles Gut! Wollte nur mal nachfragen, ob ich das Feature vielleicht einfach nur nicht finde. ;-)
